htdocs/societe/card.php

```diff
@@ -2444,10 +2444,20 @@ function init_supplier_categ() {
 		$formconfirm = '';
 
 		// Confirm delete third party
-		if ($action == 'delete' || ($conf->use_javascript_ajax && empty($conf->dol_use_jmobile))) {
-			$formconfirm = $form->formconfirm($_SERVER["PHP_SELF"]."?socid=".$object->id, $langs->trans("DeleteACompany"), $langs->trans("ConfirmDeleteCompany"), "confirm_delete", '', 0, "action-delete");
+		if ($action == 'delete') {
+			$formquestion = array(
+				array(
+					'label' => $langs->trans('M456465645y'),
+					'value' => 'yes',
+				)
+			);
+
+			$formconfirm .= $form->formconfirm($_SERVER["PHP_SELF"].'?id='.$object->id, $langs->trans('DeleteThirdParty'), $langs->trans('ConfirmDeleteCompany'), 'confirm_delete', '', 0, 2);
```

Comment:
Why DeleteACompany become DeleteThirdParty ?

Reply:
replace to "DeleteACompany"
DeleteThirdParty doesn't exist in companies.lang